### PR TITLE
fix scope compile from to runtime in shell of making wtp #437

### PR DESCRIPTION
**Pull request by btkobayashirun**
Please review #437.

### CODE REVIEW

File: unmaven-project.sh [hunk 1]
```diff
@@ -29,9 +29,9 @@ BUILDXML=build.xml
 
 download_jars() {
     rm -rf $WEBINFLIB $LIBSRC $TESTLIB
-    mvn dependency:copy-dependencies -DoutputDirectory=$WEBINFLIB -DincludeScope=compile -DexcludeScope=provided -DexcludeTypes=pom
+    mvn dependency:copy-dependencies -DoutputDirectory=$WEBINFLIB -DincludeScope=runtime -DexcludeScope=provided -DexcludeTypes=pom
     mvn dependency:copy-dependencies -DoutputDirectory=$LIBSRC -DincludeScope=compile -DexcludeScope=provided -DexcludeTypes=pom -Dclassifier=sources
-    mvn dependency:copy-dependencies -DoutputDirectory=$TESTLIB -DexcludeScope=compile -DexcludeTypes=pom
+    mvn dependency:copy-dependencies -DoutputDirectory=$TESTLIB -DexcludeScope=runtime -DexcludeTypes=pom
     mvn dependency:copy-dependencies -DoutputDirectory=$LIBSRC -DexcludeScope=compile -DexcludeTypes=pom -Dclassifier=sources
```
Comment (yoshikawaa) [inline unmaven-project.sh:35]: - [ ] This line gets source codes of the library stored in `$TESTLIB`.
You need to fix this like the above line.

Or you can put the copy to `$LIBSRC` into one line.

|  | compile | provoded | runtime | test | scope options |
|-|:-:|:-:|:-:|:-:|-|
| $WEBINFLIB | Y | N | Y | N | includeScope=runtime |
| $TESTLIB | N | Y | N | Y | excludeScope=runtime |
| $LIBSRC | Y | Y | Y | Y |  |

File: unmaven-project.sh [hunk 2]
```diff
@@ -29,9 +29,9 @@ BUILDXML=build.xml
 
 download_jars() {
     rm -rf $WEBINFLIB $LIBSRC $TESTLIB
-    mvn dependency:copy-dependencies -DoutputDirectory=$WEBINFLIB -DincludeScope=compile -DexcludeScope=provided -DexcludeTypes=pom
+    mvn dependency:copy-dependencies -DoutputDirectory=$WEBINFLIB -DincludeScope=runtime -DexcludeScope=provided -DexcludeTypes=pom
```
[the remark this answers is not in the document]
Comment (btkobayashirun) [reply unmaven-project.sh]: I fixed it with an 85ebb514d751621fc54be494c6f560bfeb2e96a5

File: unmaven-project.sh [hunk 3]
```diff
@@ -29,9 +29,9 @@ BUILDXML=build.xml
...
     mvn dependency:copy-dependencies -DoutputDirectory=$LIBSRC -DincludeScope=compile -DexcludeScope=provided -DexcludeTypes=pom -Dclassifier=sources
```
[the remark this answers is not in the document]
Comment (btkobayashirun) [reply unmaven-project.sh]: I fixed it with an e2aefd0bc0ca7fbe84e439fcbaa249df51c53f28

File: unmaven-project.sh (hunk 1 above)
Comment (btkobayashirun) [reply unmaven-project.sh]: remove option of `excludeScope`
a363cf6ec8af17bcbf148d397248b43dab6bfff9

Comment (yoshikawaa) [reply unmaven-project.sh]: You got the same Jar twice due to your wrong fix.

Comment (btkobayashirun) [reply unmaven-project.sh]: Removed duplicate command.
c119c1294bbf376a3a1f375d7d730dd8e47e203f

File: unmaven-project.sh [hunk 4]
```diff
@@ -29,10 +29,9 @@ BUILDXML=build.xml
 
 download_jars() {
     rm -rf $WEBINFLIB $LIBSRC $TESTLIB
```
Comment (yoshikawaa) [inline unmaven-project.sh:31]: It is better to arrange in the order of `$WEBINFLIB`, `$TESTLIB`, `$LIBSRC` according to the correction below.

Comment (btkobayashirun) [reply unmaven-project.sh]: Fix in 26eaf29cc1d4292b009180e7dfc984d272bd167c

**Pull request merged**